roxctl/deployment/check/check.go

```diff
@@ -116,6 +117,13 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c.Flags().BoolVar(&deploymentCheckCmd.force, "force", false, "bypass Central's cache for images and force a new pull from the Scanner")
 	utils.Must(c.MarkFlagRequired("file"))
 	c.Flags().StringVar(&deploymentCheckCmd.cluster, "cluster", "", "cluster name or ID to use as context for evaluation")
+	c.Flags().StringVarP(&deploymentCheckCmd.namespace, "namespace", "n", "default", "namespace to evaluate, defaults to \"default\" if unset.")
+	namespace, err := c.Flags().GetString("namespace")
+	// this error should never occur, it would only occur if default values are invalid
+	utils.Must(err)
+	if namespace == "default" {
+		fmt.Printf("Warning: The namespace defaulted to \"default\", if you want to check deployments in a custom namespace set it with '%s [X]' or '-%s [X]'\n", c.Flag("namespace").DefValue, c.Flag("namespace").Shorthand)
```

Comment:
what `[X]` means?

Reply:
A variable representing the namespace, I'll change it to `[NAMESPACE]`